spec/support/features/in_person_helper.rb

```diff
@@ -29,6 +29,23 @@ def fill_out_state_id_form_ok
     fill_in t('in_person_proofing.form.state_id.state_id_number'), with: GOOD_STATE_ID_NUMBER
   end
 
+  def fill_out_state_id_form_ok_include_address
```

Comment:
Instead of adding a new function could we instead add an argument to the one that already exists, like this?
```
def fill_out_state_id_form_ok(include_address: false)
    ...
    if include_address
        ... # new fields here
    end
```